base/ubi9/entrypoint.sh

```diff
@@ -5,6 +5,16 @@ if [ ! -d "${HOME}" ]; then
   mkdir -p "${HOME}"
 fi
 
+# Configure container builds to use vfs or fuse-overlayfs
+if [ ! -d "${HOME}/.config/containers" ]; then
```

Comment:
What happens if persistent home is enabled and fuse-overlayfs gets disabled? Would `${HOME}/.config/containers/storage.conf` need to be rewritten? Would the outer `if` block stop that from happening?

Reply:
The only way to disable fuse-overlayfs would be to apply a machine-config to the cluster, so I would say it's not likely.  And if it did happen, that would be a disruptive change.

The reason that I put the first `if` block there is to not overwrite any existing podman config.  The user might have added addition config to storage.conf which we would not want to clobber on a workspace restart.

Reply:
The `if` block could be a bit more targeted to look specifically for storage.conf I suppose.

```bash
if [ ! -f "${HOME}/.config/containers/storage.conf" ]; then
```

But, there are valid configs where there is no `storage.conf` file, but other content under `${HOME}/.config/containers`.  Which is why I just check for the existence of the directory, not the file.

Reply:
@cgruver  that makes sense, thanks!

Reply:
__Note:__ As of OCP 4.15 `/dev/fuse` is available for leaking into a container OOTB.  No cluster change required.

With OCP 4.15 `/dev/net/tun` is also available.  

Pod annotation: `io.kubernetes.cri-o.Devices: "/dev/fuse,/dev/net/tun"`

So, to disable `fuse-overlayfs` support would require disruptive intervention on the part of cluster admins.